### PR TITLE
Intel OpenCL Runtime for Testing

### PR DESCRIPTION
**Pull request by tremmelg**
Install the Intel OpenCL CPU Runtime and also run the tests using it.

### CODE REVIEW

**Comment by Ruyk** [comment]
Amazing! This will improve the testing in the open source repositories. Can you print the output of ComputeCpp info on the travis log and also remove the verbose wget to avoid having a massive log?

File: .travis/install_intel_opencl.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ev
+
+###########################
+# Get Intel OpenCL Runtime
+###########################
+
+wget http://registrationcenter-download.intel.com/akdlm/irc_nas/12513/opencl_runtime_16.1.2_x64_rh_6.4.0.37.tgz -O /tmp/opencl_runtime.tgz
```
Comment (Ruyk) [inline .travis/install_intel_opencl.sh]: Note that the version number or package name may change. Can you put the name and the URL on separate variables ?

File: Dockerfile [hunk 2]
```diff
@@ -34,19 +35,27 @@ RUN apt-get install -yq --allow-downgrades --allow-remove-essential           \
...
 
 CMD cd /SyclParallelSTL && \
     if [ "${SYCL_IMPL}" = 'triSYCL' ]; then \
       ./build.sh --trisycl -DTRISYCL_INCLUDE_DIR=/tmp/triSYCL-master/include; \
     elif [ "${SYCL_IMPL}" = 'COMPUTECPP' ]; then \
-      COMPUTECPP_TARGET="host" ./build.sh /tmp/ComputeCpp-latest; \
+      if [ "${TARGET}" = 'host' ]; then \
+        COMPUTECPP_TARGET="host" ./build.sh /tmp/ComputeCpp-latest; \
+      else \
+        ./build.sh /tmp/ComputeCpp-latest; \
```
Comment (Ruyk) [inline Dockerfile]: to be on the safe side, use COMPUTECPP_TARGET="intel:cpu" here

**Comment by tremmelg** [comment]
Output from `computecpp_info`
https://travis-ci.org/KhronosGroup/SyclParallelSTL/jobs/319745095#L4844

File: .travis/install_intel_opencl.sh (hunk 1 above)
Comment (tremmelg) [reply .travis/install_intel_opencl.sh]: Is this what you meant?
https://github.com/KhronosGroup/SyclParallelSTL/pull/49/commits/01771ea41018af928df323ba04134858ff820158

File: Dockerfile (hunk 2 above)
Comment (tremmelg) [reply Dockerfile]: Done.

**Comment by Ruyk** [comment]
That looks good to me! Good work

**Pull request merged**